_posts/2024-02-07-#28950.md

```diff
@@ -8,16 +8,50 @@ authors: [instagibbs]
 components: ["rpc"]
 host: ismaelsadeeq
 status: upcoming
-commit:
+commit: bf6fbea9794db5e8d0adce8ff6182574808cea54
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
+- The [`submitpackage RPC`](https://bitcoincore.org/en/doc/26.0.0/rpc/rawtransactions/submitpackage/) RPC method takes `package` as a required argument. The package must consist of a child with its parents, and none of the parents may depend on one another. The package will be validated according to consensus and mempool policy rules. If all transactions pass, they will be accepted to mempool.
```

Comment:
Why copy the docs here? Just the link is fine.

Reply:
Yes fixed 👍🏾